### ISSUE TITLE
Astropy issues deprecation warning with CompImageHDU

### ISSUE THREAD

**Issue by timj** [LSSTDESC/healsparse#141]
```python
healsparse/fits_shim.py:236: AstropyDeprecationWarning: The tile_size argument has been deprecated. Use tile_shape instead, but note that this should be given in the reverse order to tile_size (tile_shape should be in Numpy C order).
   hdu = fits.CompImageHDU(data=sparse_map, header=fits.Header(),
```

Looks like it was deprecated in astropy 5.3 in astropy/astropy#14428

**Comment by erykoff** [comment]
So I need a version check to support older astropy.  Or a try/except with the newer API.  Lovely.

**Comment by erykoff** [comment]
Fixed on #142
